package.json

```diff
@@ -6,7 +6,7 @@
     "build": "npm run build:scss && npm run build:iconsets",
     "build:scss": "node ./cli/create-scss-cli -o icons.scss images/**/*.*",
     "build:iconsets": "node ./cli/create-iconset-cli images/tier1 images/tier2 images/tier3 images/html-editor images/emoji",
-    "postinstall": "bower install",
+    "postinstall": "polymer install",
```

Comment:
Why not `polymer install --variants` and not need to put that line in the `.travis.yml` file?